### PR TITLE
chore: remove deprecated plugins that have been migrated to community-plugins

### PR DESCRIPTION
**Pull request by 04kash**
remove 3scale-backend, acr, argocd and argocd-common as they have been migrated to [backstage/community-plugins](https://github.com/backstage/community-plugins)

### CODE REVIEW

**Review by JessicaJHee — DISMISSED** [review]
There are a few more instances of these plugins being mentioned that can be removed, such as [here](https://github.com/janus-idp/backstage-plugins/blob/3ff7c593a14c371a5d42059d246717d474a9cc9d/catalog-info.yaml#L46-L47) and [here](https://github.com/janus-idp/backstage-plugins/blob/3ff7c593a14c371a5d42059d246717d474a9cc9d/.github/CODEOWNERS#L45)

**Comment by 04kash** [comment]
I've removed them from `catalog-info.yaml` but I think we should still keep the `CODEOWNERS` file as is for future reference

**Comment by JessicaJHee** [comment]
@04kash there are still some references in [this file](https://github.com/janus-idp/backstage-plugins/blob/main/.github/renovate.json), but should be good to merge after this!

**Comment by 04kash** [comment]
> @04kash there are still some references in [this file](https://github.com/janus-idp/backstage-plugins/blob/main/.github/renovate.json), but should be good to merge after this!

Done!

File: catalog-info.yaml
Comment (nickboldt) [inline catalog-info.yaml:48]: file not found at https://github.com/janus-idp/backstage-plugins/tree/main/plugins/dynamic-plugins-info/catalog-info.yaml ... is it somewhere else?

Comment (04kash) [reply catalog-info.yaml]: Removed it, wasn't there in the catalog-info.yaml before, don't know how it got added

**Review by nickboldt — APPROVED** [review]

**Comment by nickboldt** [comment]
tests failing, yarn.lock dirty

**Review by JessicaJHee — APPROVED** [review]
Thanks for addressing everything! lgtm after the test/build succeeds 👍

**Review by JessicaJHee — APPROVED** [review]

**Pull request merged**
